src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+:root {
+  --light-shadow: rgba(61, 78, 97, 0.1);
+  --dark-shadow: rgba(61, 78, 97, 0.2);
+  --gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", Verdana, sans-serif;
+}
+
+.form {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin: 20px 0;
+}
+
+.form--small {
+  height: 42px;
+}
+
+.form__searchbar {
+  position: relative;
+  width: 100%;
+}
+
+.form__input {
+  font-family: inherit;
+  width: 100%;
+  height: 100%;
+  padding: 25px 0 25px 62px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 var(--light-shadow);
+  font-size: 16px;
+}
+
+.form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.form__input--small {
+  padding: 11px 0 11px 33px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.form__input--small::placeholder {
+  font-size: 14px;
+}
+
+.form__icon {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  top: 25px;
```

Comment:
Here these values are reliant on the inputs padding and height, you could export them to a separate variable:
```css
.form {
  display: flex;
  width: 100%;
  height: var(--big-form-height);  /* this as well as the padding left could be used as variables */
  margin: 20px 0;
}

...

.form__icon {
  top: calc(var(--big-form-height) / 2);  /* now instead of calculating the values on our own, we may use the calc */
  left: calc(var(--big-padding-left) / 2); /* function and divide these values by two to place icon in the middle */
  transform: translate(-50%, -50%); /* icon positions its edge in the middle so we have to translate it by its size */
  ...
}
```